tests/testImages.cpp

```diff
@@ -82,11 +82,10 @@ BOOST_AUTO_TEST_CASE(render_demo_with_proximity_renderer)
     auto& testSuite = boost::unit_test::framework::master_test_suite();
 
     const char* app = testSuite.argv[0];
-    const char* argv[] = {app,         "--renderer",
-                          "proximity", "--samples-per-pixel",
-                          "256",       "--synchronous-mode",
-                          "demo",      "--window-size",
-                          "400",       "300"};
+    const char* argv[] = {
+        app,   "--renderer", "proximity",     "--samples-per-pixel",
+        "256", "demo",       "--window-size", "400",
+        "300"};
```

Comment:
remove extra spaces

Reply:
It's clang that does it 😄

Reply:
Yes, and it will do it again, that's why I suggest to remove the spaces and give up on the original format.

Reply:
the spaces are added (back) by clang-format...

Reply:
Actually, clang-format is making 4 columns...